src/pages/ReviewPage.tsx

```diff
@@ -356,160 +357,155 @@ class ReviewPage extends React.Component<RouteComponentProps, State> {
+            onScroll={updateContestChoicesScrollStates}
           >
-            <ScrollContainer
-              ref={this.scrollContainer}
-              onScroll={this.updateContestChoicesScrollStates}
-            >
-              <ScrollableContentWrapper isScrollable={isScrollable}>
-                {(contests as Contests).map((contest, i) => (
-                  <Contest
-                    id={`contest-${contest.id}`}
-                    key={contest.id}
-                    to={`/contests/${i}#review`}
-                  >
-                    <ContestProse compact>
-                      <h2 aria-label={`${contest.title.replace(',', '')},`}>
-                        <ContestSection>{contest.section}</ContestSection>
-                        {contest.title}
-                      </h2>
-
-                      {contest.type === 'candidate' && (
-                        <CandidateContestResult
-                          contest={contest}
-                          parties={parties}
-                          vote={votes[contest.id] as CandidateVote}
-                        />
-                      )}
-                      {contest.type === 'yesno' && (
-                        <YesNoContestResult
-                          contest={contest}
-                          vote={votes[contest.id] as YesNoVote}
-                        />
-                      )}
-                      {contest.type === 'ms-either-neither' && (
-                        <MsEitherNeitherContestResult
-                          contest={contest}
-                          eitherNeitherContestVote={
-                            votes[
-                              contest.eitherNeitherContestId
-                            ] as OptionalYesNoVote
-                          }
-                          pickOneContestVote={
-                            votes[contest.pickOneContestId] as OptionalYesNoVote
-                          }
-                        />
-                      )}
-                    </ContestProse>
-                    <ContestActions aria-label="press enter to change your answer for this contest.">
-                      <DecoyButton primary aria-hidden>
-                        Change
-                      </DecoyButton>
-                    </ContestActions>
-                  </Contest>
-                ))}
-              </ScrollableContentWrapper>
-            </ScrollContainer>
-            {isScrollable /* istanbul ignore next: Tested by Cypress */ && (
-              <ScrollControls aria-hidden>
-                <Button
-                  className="scroll-up"
-                  big
-                  primary
-                  aria-hidden
-                  data-direction="up"
-                  disabled={isScrollAtTop}
-                  onPress={this.scrollContestChoices}
+            <ScrollableContentWrapper isScrollable={isScrollable}>
+              {(contests as Contests).map((contest, i) => (
```

Comment:
Keeping this as it was is fine for now, but really this cast is unneeded:

```suggestion
              {contests.map((contest, i) => (
```

Reply:
thx. done.